lib/idseq-dag/idseq_dag/steps/blast_contigs.py

```diff
@@ -482,8 +485,12 @@ def generate_contig_taxon_summary(
         _get_top_m8_nr(m8_file, top_m8_output)
 
     read_dict, accession_dict, _ = m8.summarize_hits(hit_summary)
+    read2contig = {}
+    read2contig_bases = {}
     with open(read_to_contig_tsv_path) as f:
-        read2contig = {row[0]: row[1] for row in csv.reader(f, delimiter="\t")}
+        for row in csv.reader(f, delimiter="\t"):
```

Comment:
I know I did this in the first place but can you variable name these? Like

```python
for read_id, contig_id, contig in csv.reader(f, delimiter="\t"):
```

It makes it a bit easier to track what is going on below.